torch/csrc/jit/python/pybind_utils.cpp

```diff
@@ -114,6 +114,16 @@ IValue toIValue(py::handle obj, const TypePtr& type, c10::optional<int32_t> N) {
       if (torch::is_symfloat(py::handle(obj))) {
         return py::cast<c10::SymFloat>(obj).guard_float(__FILE__, __LINE__);
       }
+      if (THPVariable_Check(obj.ptr())) {
+        auto var = py::cast<autograd::Variable>(obj);
+        // NB: We carefully test if the storage is meta, because that is
+        // always accurate even if you have a fake tensor (which is the
+        // primary case we are trying to detect here)
+        if (var.storage().device_type() == kMeta) {
```

Comment:
I was hitting a build failure when building this branch locally. Writing `c10::kMeta` fixes it. It should also fix the macos error you are seeing now.